services/agents/registry.go

```diff
@@ -1004,6 +1004,36 @@ func (r *Registry) StartPTSummaryAction(ctx context.Context, id, pmmAgentID stri
 	return nil
 }
 
+// StartPTMySQLSummaryAction starts pt-mysql-summary action on the pmm-agent.
+// The pt-mysql-summary's execution may require some of the following params: IP address, port, socket, username, password.
+// The function returns nil if ok, otherwise an error code
+func (r *Registry) StartPTMySQLSummaryAction(ctx context.Context, id, pmmAgentID, address string, port uint16, socket, username, password string) error {
+	// Action request data that'll be sent to agent
+	actionRequest := &agentpb.StartActionRequest{
+		ActionId: id,
+		// Proper params that'll will be passed to the command on the agent's side, even empty, othervise request's marshal fail.
```

Comment:
it' not true. 
```suggestion
		// Proper params that'll will be passed to the command on the agent's side.
```
in PTSummary it was added because we passed empty params.